OneCommunityGlobal/HighestGoodNetworkApp
# Chetan fix margin for dashboard card

# Description
Fixed the margin for popup card on the dashboard

## Related PRS (if any):
This frontend PR is related to the development backend.

## Main changes explained:
- Added the CSS logic to Header.css file

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Log in as volunteer/admin user
5. Change the date on your pc to the recent thursday
6. Verfiy the below UI differences
7. verify this new feature works in [dark mode](https://docs.google.com/document/d/11OXJfBBedK6vV-XvqWR8A9lOH0BsfnaHx01h1NZZXfI)

## Screenshots or videos of changes:
![Screenshot 2024-12-26 122354](https://github.com/user-attachments/assets/c2bf8319-cde5-4378-bfe5-2fe280407fe4)
![WhatsApp Image 2024-12-20 at 12 44 25 PM (1)](https://github.com/user-attachments/assets/d2b2b80c-d811-4e01-b0cd-fea785f433bd)

## Note:
Include the information the reviewers need to know.
